cypress/e2e/Navbar.spec.ts

```diff
@@ -16,18 +16,22 @@ describe('<Navbar />', () => {
     it('Opens and closes the MapGuide', () => {
         cy.visit('/');
 
-        cy.get('.MuiDialog-paperScrollPaper').should('not.exist');
+        cy.get('.MuiTooltip-tooltip').should('not.exist');
         cy.contains(/Map Guide/i)
             .should('be.visible')
-            .click();
-        cy.get('.MuiDialog-paperScrollPaper').should('exist');
-        cy.get('[aria-label="close"]').click();
-        cy.get('.MuiDialog-paperScrollPaper').should('not.exist');
+            .trigger('mouseover');
+        cy.get('.MuiTooltip-tooltip').should('exist');
         cy.contains(/Map Guide/i)
             .should('be.visible')
-            .click();
-        cy.get('body').click(0, 0);
-        cy.get('.MuiDialog-paperScrollPaper').should('not.exist');
+            .trigger('mouseout');
+        cy.get('.MuiTooltip-tooltip').should('not.exist');
+
+        cy.contains(/Map Guide/i)
```

Comment:
I think that just the code starting from this section can be included in the test, the first half of the test basically checks the same thing